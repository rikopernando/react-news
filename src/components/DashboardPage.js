import React from 'react'
import NewsList from './NewsList'

const DashboardPage = () => (
  <div>
    <div className="content-container">
      <h2>
        Indonesia news 
      </h2> 
      <hr />
    </div>
    <NewsList />
  </div>
)

export default DashboardPage

