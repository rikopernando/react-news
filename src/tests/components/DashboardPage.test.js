import React from 'react'
import { shallow } from 'enzyme'
import DashboardPage from '../../components/DashboardPage'

test('should render DashboardPage correctly', () => {
  const wrapper = shallow(<DashboardPage />)
  expect(wrapper).toMatchSnapshot()
  /*
  const renderer = new ReactShallowRenderer()
  renderer.render(<Header />)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
  */
})
