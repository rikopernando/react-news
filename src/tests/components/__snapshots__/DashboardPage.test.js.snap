// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage correctly 1`] = `
<div>
  <div
    className="content-container"
  >
    <h2>
      Indonesia news
    </h2>
    <hr />
  </div>
  <Connect(NewsList) />
</div>
`;
